homeworks/vadym.myroshnyk_VadimMir/homework_7/homework_7.test.js

```diff
@@ -0,0 +1,37 @@
+const fib = require('./homework_7');
+
+describe('fibonacci', () => {
+    test('check if returned value is array', () => {
+        expect(Array.isArray(fib())).toBe(true);
+    });
+
+    test('check Fibonacci sequence until max number of 50', () => {
+        const array = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34];
+        const arrayFinonacci = fib(50);
+        for (let i = 0; i < arrayFinonacci.lenght; i++) {
+            expect(arrayFinonacci[i]).toEqual(arrayFinonacci[i] === array[i]);
+        }
+    });
+
+    test('check Fibonacci sequence until max number of 100', () => {
+        const array = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89];
+        const arrayFinonacci = fib(50);
+        for (let i = 0; i < arrayFinonacci.lenght; i++) {
+            expect(arrayFinonacci[i]).toEqual(arrayFinonacci[i] === array[i]);
+        }
```

Comment:
`for (let i = 2; i < arrayFinonacci.lenght; i++) {`
    `expect(arrayFinonacci[i]).toEqual(arrayFinonacci[i - 1] + arrayFinonacci[i - 1]);`
`}`
maybe this will make more sense? very simple and straightforward